pages/community/dei-mission-statement.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: DEI Mission Statement
+subtitle: Diversity, Equality, and Inclusion at US-RSE
+bigimg:
```

Comment:
```suggestion
```

Reply:
This issue also exists in the "mission.md" file. I'm going to change it there as well.